bigframes/operations/__init__.py

```diff
@@ -415,6 +416,15 @@ def output_type(self, *input_types):
         return input_types[0]
 
 
+@dataclasses.dataclass(frozen=True)
+class StrftimeOp(UnaryOp):
+    name: typing.ClassVar[str] = "strftime"
+    date_format: str
+
+    def output_type(self, *input_types):
+        return input_types[0]
```

Comment:
output type will always be string. best to use `bigframes.dtypes.STRING_DTYPE` in case we switch our default string dtype

Reply:
Sounds good.